package.json

```diff
@@ -30,8 +30,7 @@
     "gulp-nsp": "^0.4.5",
     "gulp-babel": "^5.1.0",
     "babel-core": "^5.5.0",
-    "nock": "*",
-    "locus": "*"
```

Comment:
It doesn't appear to be used anywhere, and its dependency is failing on node 4.0